marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/MarathonWorkerRunTask.kt

```diff
@@ -0,0 +1,12 @@
+package com.malinskiy.marathon
+
+import com.malinskiy.marathon.worker.MarathonWorker
+import org.gradle.api.DefaultTask
+import org.gradle.api.tasks.TaskAction
+
+abstract class MarathonWorkerRunTask : DefaultTask() {
+    @TaskAction
+    fun run() {
+        MarathonWorker.stop()
```

Comment:
Naming isn't good - we are stopping worker in task that has WorkerRun in the name. Maybe rename it to MarathonTerminationTask / MarathonWorkerStopTask / MarathonFinalizationTask?